jbpm-benchmarks/jbpm-performance-tests-jmh/src/main/filtered-resources/META-INF/persistence.xml

```diff
@@ -75,4 +76,84 @@
     </properties>
   </persistence-unit>
 
-</persistence>    
+  <persistence-unit name="org.jbpm.domain" transaction-type="JTA">
```

Comment:
Problem with this is that this PU is hardcoded already in here:
https://github.com/kiegroup/jbpm/blob/master/jbpm-test/src/main/java/org/jbpm/test/services/AbstractKieServicesTest.java#L108
which is the class I extend from.
However it makes sense to me but at the same time it might behave differently from how the rest of the test (in jbpm at least) are working now by making this dynamic - wdyt?

Reply:
Yes, agree. We can solve it later.